installing/installing_vsphere/installing-restricted-networks-vsphere.adoc

```diff
@@ -66,6 +66,8 @@ include::modules/installation-extracting-infraid.adoc[leveloffset=+1]
 
 include::modules/installation-vsphere-machines.adoc[leveloffset=+1]
 
+You can add compute machines after the cluster installation is completed by following xref:../../machine_management/user_infra/adding-vsphere-compute-user-infra.adoc#adding-vsphere-compute-user-infra[Adding compute machines to vSphere].
+
```

Comment:
> "You can add compute machines after the cluster installation is completed"
I think it meant to add extra compute machines after the cluster installation, right? Then how about place this comment after the section of installation complete, https://docs.openshift.com/container-platform/4.6/installing/installing_vsphere/installing-restricted-networks-vsphere.html#installation-complete-user-infra_installing-restricted-networks-vsphere? Because after [installation-vsphere-machines.adoc], the cluster installation does not complete yet.

Reply:
@jiajliu I added that text there because the reported bug was related to the admonition after step 8 [here](https://docs.openshift.com/container-platform/4.6/installing/installing_vsphere/installing-restricted-networks-vsphere.html#installation-vsphere-machines_installing-restricted-networks-vsphere).
> You must create the bootstrap and control plane machines at this time. Because some pods are deployed on compute machines by default, also create at least two compute machines before you install the cluster.

This added text would be directly after that admonition, which is instructing the user to create at least two compute machines before installing the cluster. So really, my added text should have read
>You can add compute machines ~~after the cluster installation is completed~~ by following [Adding compute machines to vSphere](https://docs.openshift.com/container-platform/4.6/machine_management/user_infra/adding-vsphere-compute-user-infra.html).

Would the proposed text be correct if that change was made?

Reply:
@codyhoag If it's for adding two compute machines after step8, then following Adding compute machines to vSphere[https://docs.openshift.com/container-platform/4.6/machine_management/user_infra/adding-vsphere-compute-user-infra.html](url) should be confused because the "Prerequisites" and "Approving the CSRs for your machines" steps are not correct at this step. It's better to refer to Creating more Red Hat Enterprise Linux CoreOS (RHCOS) machines in vSphere[https://docs.openshift.com/container-platform/4.6/machine_management/user_infra/adding-vsphere-compute-user-infra.html#machine-vsphere-machines_adding-vsphere-compute-user-infra](url) directly. HDYT?

Reply:
@jiajliu yeah, good points. I'll move this after the [Completing installation on user-provisioned infrastructure](https://docs.openshift.com/container-platform/4.6/installing/installing_vsphere/installing-restricted-networks-vsphere.html#installation-complete-user-infra_installing-restricted-networks-vsphere) as you originally suggested :+1: